im-sdk-core/src/main/java/com/easemob/im/server/api/sms/SmsApi.java

```diff
@@ -24,6 +29,7 @@ public SmsApi(Context context) {
      * @param extendCode  扩展码
      * @param custom  用户自定义属性，长度不能超过64字符
```

Comment:
What do they mean? Need some 'loud' names and comments here. Again, we should handle & hide underlying things from developers.
- tid
- tmap
- extendCode
- custom